lib/terminus.spec.js

```diff
@@ -658,25 +658,37 @@ describe('Terminus', () => {
   })
 
   it('accepts custom headers', async () => {
-  
-  createTerminus(server, {
-    healthChecks: {
-      '/health': () => {
-        return Promise.resolve()
-      }
-    },
-    headers: {
-      "Access-Control-Allow-Origin": "*",
-      "Access-Control-Allow-Methods": "OPTIONS, POST, GET",
-    },
+    createTerminus(server, {
+      healthChecks: {
+        '/health': () => {
+          return Promise.resolve()
+        }
+      },
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Allow-Methods": "OPTIONS, POST, GET",
+      },
+    })
+    server.listen(8000)
+
+    const res = await fetch('http://localhost:8000/health')
+    expect(res.status).to.eql(200)
+    expect(res.headers.has('Access-Control-Allow-Methods')).to.eql(true)
+    expect(res.headers.get('Access-Control-Allow-Methods')).to.eql('OPTIONS, POST, GET')
+    expect(res.headers.has('Access-Control-Allow-Origin')).to.eql(true)
+    expect(res.headers.get('Access-Control-Allow-Origin')).to.eql('*')
+  })
+
+  it('allows to exit(0) with useExit0', async () => {
+  let responseAssertionsComplete = false
+
+  // We're only truly finished when the response has been analyzed and the forked http process has exited,
+  // freeing up port 8000 for future tests
+  execFile('node', ['lib/standalone-tests/terminus.useExit0.js'], (error) => {
+    fail(`exit code was non zero ${error.signal}`)
   })
-  server.listen(8000)
-
-  const res = await fetch('http://localhost:8000/health')
-  expect(res.status).to.eql(200)
-  expect(res.headers.has('Access-Control-Allow-Methods')).to.eql(true)
-  expect(res.headers.get('Access-Control-Allow-Methods')).to.eql('OPTIONS, POST, GET')
-  expect(res.headers.has('Access-Control-Allow-Origin')).to.eql(true)
-  expect(res.headers.get('Access-Control-Allow-Origin')).to.eql('*')
+  .then(res => {
+    // FIXME: Check if exit code was 0
+    // expect()
```

Comment:
can you fix the ident here please? and also add the expectation that you started on line 692?

Reply:
@gergelyke actually I asked you for help in the previous PR. So maybe you can give me a hint on how to get the exit code from the execFile function?

Reply:
Ok fixed